Ska/Sun.py

```diff
@@ -205,3 +250,29 @@ def off_nominal_roll(att, time):
         off_nom_roll -= 360
 
     return off_nom_roll
+
+
+def allowed_rolldev(pitch):
+    """
+    Determine the maximum allowed deviation from nominal roll, in degrees.
+
+    :param pitch: spacecraft pitch in degrees (can be from Ska.Sun.pitch)
+    :returns: maximum roll deviation / difference from nominal roll
+    """
+    idx = np.searchsorted(ROLL_TABLE['pitch'], pitch, side='right')
+    return ROLL_TABLE['rolldev'][idx - 1]
```

Comment:
This presently doesn't return really meaningful results for not-allowed pitch between 0 and 46.  Should probably return error instead of just returning 19.99.